src/Sylius/Component/Order/Model/Order.php

```diff
@@ -271,6 +271,11 @@ public function getState(): string
      */
     public function setState(string $state): void
     {
+        @trigger_error(
```

Comment:
`setState` is used by state machine itself so I'm not sure we can deprecate it now (unless we provide another way for the state machine to change it).